tests/child_chain/test_child_chain.py

```diff
@@ -1,26 +1,31 @@
 import rlp
 import pytest
+from unittest.mock import patch, MagicMock
 from ethereum import utils as u
 from plasma.child_chain.transaction import Transaction
 from plasma.child_chain.block import Block
 from plasma.child_chain.child_chain import ChildChain
 
-
-key1 = u.normalize_key(b'8b76243a95f959bf101248474e6bdacdedc8ad995d287c24616a41bd51642965')
-invalid_key = u.normalize_key(b'8a76243a95f959bf101248474e6bdacdedc8ad995d287c24616a41bd51642965')
+AUTHORITY = b'\xfd\x02\xec\xeeby~u\xd8k\xcf\xf1d.\xb0\x84J\xfb(\xc7'
+tx_key = u.normalize_key(b'8b76243a95f959bf101248474e6bdacdedc8ad995d287c24616a41bd51642965')
+invalid_tx_key = u.normalize_key(b'8a76243a95f959bf101248474e6bdacdedc8ad995d287c24616a41bd51642965')
+block_key = u.normalize_key(b'3bb369fecdc16b93b99514d8ed9c2e87c5824cf4a6a98d2e8e91b7dd0c063304')
+invalid_block_key = u.normalize_key(b'3ab369fecdc16b93b99514d8ed9c2e87c5824cf4a6a98d2e8e91b7dd0c063304')
 newowner1 = b'\x8cT\xa4\xa0\x17\x9f$\x80\x1fI\xf92-\xab<\x87\xeb\x19L\x9b'
 amount1 = 200
 amount2 = 400
 
 
 @pytest.fixture
-def root_chain(get_contract):
-    return get_contract('RootChain/RootChain.sol')
-
-
-@pytest.fixture
-def child_chain(root_chain):
-    child_chain = ChildChain(None, None, None)
+def child_chain():
+    with patch('plasma.child_chain.child_chain.ChildChain.__init__', return_value=None) as child_chain:
```

Comment:
I guess we don't need the patch and 
```python
child_chain = ChildChain(None, None, None)
child_chain.root_chain = Mock()
```
should be enough?

From what I read is that you patch the `__init__` function as `child_chain`, but soon make `child_chain` become `ChildChain(None, None, None)`, so the patch is probably not used.
And since probably we don't need any magic value from the `root_chain`, `Mock` should probably be enough.